amlb/benchmarks/openml.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from typing import List, Tuple, Optional
+
+import openml
+
+from amlb.utils import Namespace
+
+
+log = logging.getLogger(__name__)
+
+
+def is_openml_benchmark(benchmark: str) -> bool:
+    """ Check if 'benchmark' is a valid identifier for an openml task or study. """
+    if len(benchmark.split('/')) == 3:
+        domain, oml_type, oml_id = benchmark.split('/')
+        supported_types = ['s', 't']
+
+        valid_id = False
+        try:
+            _ = int(oml_id)
+            valid_id = True
+        except ValueError:
+            pass
```

Comment:
```suggestion
valid_id = oml_id.isdecimal()
```
should do the job

Reply:
I wasn't sure - the numbers need to be integer, otherwise the openml-python api will raise an error later. I suppose this is wrong too since it does not enforce non-zero positive integers. Will rewrite.